packages/dev/codemods/src/s1-to-s2/__tests__/table.test.ts

```diff
@@ -6,9 +6,92 @@ const test = (name: string, input: string) => {
   defineSnapshotTest(transform, {}, input, name);
 };
 
-test('Renames variants', `
-// import {TableView, TableHeader, TableBody, Column, Row, Cell} from '@adobe/react-spectrum';
+test('Static TableView', `
+import {Cell, Column, Row, TableBody, TableHeader, TableView}  from '@adobe/react-spectrum';
 
-<div>
-</div>
+<TableView>
+  <TableHeader>
+    <Column key="test">Test</Column>
+    <Column title="Blah">
+      <Column title="Group 1">
```

Comment:
I think RAC/S2 doesn't support nested columns yet.

Reply:
Will leave a leave a comment when this is found for now: [a752dea](https://github.com/adobe/react-spectrum/pull/7117/commits/a752dea3b45c702e72e53e2b648aaf0344253556)